### DISCUSSION TITLE
Beatmaps unable to load and unable to login

### DISCUSSION THREAD

**Discussion by SUPERDERP12362** [ppy/osu#21131] [Q&A]
any help would be appreciated

**Comment by Axilotl17** [comment]
we need more info. what error message is displayed upon trying to login? does loading older versions fix it? if you check in your osu! folder, do you have a corrupted client.realm? we need you to say more than just the title.

**Reply by SUPERDERP12362** [reply]
it just says failed to load

**Reply by SUPERDERP12362** [reply]
when i click a download button 
was able to log in tho

**Reply by Axilotl17** [reply]
does it pop up in a notification and say "failed to load"? and did you try using an older version of osu?

**Reply by SUPERDERP12362** [reply]
how do i do that and yes

**Comment by frenzibyte** [comment]
Please attach logs after encountering the issue.
 
![CleanShot 2022-11-06 at 16 11 21](https://user-images.githubusercontent.com/22781491/200172646-c38b0689-8bbc-476a-aa7c-0b44853ae8af.png)

**Reply by SUPERDERP12362** [reply]
what log should i attach

**Reply by frenzibyte** [reply]
All logs.

**Reply by SUPERDERP12362** [reply]
[database.log](https://github.com/ppy/osu/files/9956736/database.log)
[input.log](https://github.com/ppy/osu/files/9956737/input.log)
[network.log](https://github.com/ppy/osu/files/9956738/network.log)
[performance.log](https://github.com/ppy/osu/files/9956739/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9956740/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9956742/updater.log)

**Reply by SUPERDERP12362** [reply]
anything else?

**Reply by frenzibyte** [reply]
The network logs seem to show that accessing `osu.ppy.sh` on your end works fine, while other relevant domains such as `assets.ppy.sh`, `spectator.ppy.sh`, etc. fail with the following error:
```
2022-11-06 21:15:26 [verbose]: ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure: RemoteCertificateNameMismatch, RemoteCertificateChainErrors
2022-11-06 21:15:26 [verbose]: at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)
2022-11-06 21:15:26 [verbose]: at System.Net.Security.SslStream.CompleteHandshake(SslAuthenticationOptions sslAuthenticationOptions)
2022-11-06 21:15:26 [verbose]: at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)
```

**Reply by SUPERDERP12362** [reply]
how can I fix this?

**Reply by frenzibyte** [reply]
I'm only bringing information from the logs, please wait until a solution is given to you.

**Reply by SUPERDERP12362** [reply]
sorry

**Reply by cdwcgt** [reply]
Do you have some kind of proxy turned on or ever changed the certificate configuration?

can you visit assets.ppy.sh directly from your browser?

**Reply by ThePooN** [reply]
Sounds like an issue on your end (something tampering with your network/DNS).

Please try visiting the following addresses using MS Edge:
- https://osu.ppy.sh/home (osu! homepage)
- https://assets.ppy.sh/beatmaps/1650817/covers/cover.jpg (beatmap banner)
- https://spectator.ppy.sh/metadata (should display "Connection ID required")
- https://bm10.ppy.sh (should display "osu! beatmap mirror")

If possible, please also attempt routing your internet connection differently through a proxy, VPN, or mobile hotspot, or try another computer on the same network.

Please let us know the results!
